.gitignore

```diff
@@ -120,4 +120,4 @@ ENV/
 .mypy_cache/
 
 # emacs buffers
-\#*
+\#*
```

Comment:
I believe this is an unintended whitespace change.

Reply:
My IDE automatically removes white spaces at the end of lines from any file I edit.